### PR TITLE
Some small changes to nagging issues.

### PR DESCRIPTION
**Pull request by tme321**
Added `"root": true` to the eslint config so it stops throwing the same warning every time.

Added  `--frozen-lockfile` flag to the readme instructions.

Cleaned up imports in `character-aggregation-display.tsx`.

Removed `key` from `Row` in `character-aggregation-display.tsx`, `Row` is not iterated over so it doesn't need a `key` value.

Changed cursor to `cursor-pointer` in `styled-select-2.tsx` clickable elements.

Removed unnecessary fragment elements in `atlas.tsx`  and `characters.tsx`.

Added a `key` to the iterated over `tr` in `atlas.tsx`.

### CODE REVIEW

File: src/components/character-aggregation-display.tsx [hunk 1]
```diff
@@ -48,9 +46,8 @@ export default function CharacterAggregationDisplay({
     )
     .sort((a, b) => b.value - a.value);
 
-  const Row = ({ index, key, style }) => (
+  const Row = ({ index, style }) => (
```
Comment (zach-herridge) [inline src/components/character-aggregation-display.tsx:49]: I think this does need the key, it's iterated over by the the `react-window` lib to create the rows in the list.

**Review by zach-herridge — CHANGES_REQUESTED** [review]

**Comment by tme321** [comment]
I have dived deeper into [react-window](https://github.com/bvaughn/react-window) than I ever would have cared to.

None of the [examples](https://react-window.vercel.app/#/examples/list/fixed-size) specify a `key` field for the row definitions.

If we look at [how the items are actually rendered](https://github.com/bvaughn/react-window/blob/51ec3f896e7501acd6f399c759047781112fadf0/src/createListComponent.js#L343) you will see that a `key` is added by the library.  But it is done internally.

The props defined for the passed row component are defined [here](https://github.com/bvaughn/react-window/blob/51ec3f896e7501acd6f399c759047781112fadf0/src/createListComponent.js#L17) and do not contain a `key` field.

When I add `key` back in to the props my editor shows it as an error:

image

And finally, on my version without `key` specified one still gets added at runtime, seen here in the dev console:

image

I believe `key` should not be specified, this component handles it internally and does not want or expect us to add it into the rows ourselves.

**Comment by zach-herridge** [comment]
> I have dived deeper into [react-window](https://github.com/bvaughn/react-window) than I ever would have cared to.
> 
> None of the [examples](https://react-window.vercel.app/#/examples/list/fixed-size) specify a `key` field for the row definitions.
> 
> If we look at [how the items are actually rendered](https://github.com/bvaughn/react-window/blob/51ec3f896e7501acd6f399c759047781112fadf0/src/createListComponent.js#L343) you will see that a `key` is added by the library. But it is done internally.
> 
> The props defined for the passed row component are defined [here](https://github.com/bvaughn/react-window/blob/51ec3f896e7501acd6f399c759047781112fadf0/src/createListComponent.js#L17) and do not contain a `key` field.
> 
> When I add `key` back in to the props my editor shows it as an error:
> 
> image
> 
> And finally, on my version without `key` specified one still gets added at runtime, seen here in the dev console:
> 
> image
> 
> I believe `key` should not be specified, this component handles it internally and does not want or expect us to add it into the rows ourselves.

Cool nice research no idea where I got the key from good to know!

**Review by zach-herridge — APPROVED** [review]
looks good!

**Pull request merged**